src/components/workflowEditor/types.ts

```diff
@@ -18,12 +18,14 @@ export interface WorkflowEditState {
     cIMenuPosition: {
         top: number;
         left: number;
-    }
+    };
+    showSuccessScreen: boolean;
+    environmentId?: number;
 }
 
 export interface WorkflowEditProps extends RouteComponentProps<{ appId: string, workflowId: string, ciPipelineId: string, cdPipelineId: string }> {
     configStatus: number;
-    isCiPipeline: boolean;
+    isCDPipeline: boolean;
```

Comment:
Is `isCiPipeline` removal expected?

Reply:
That step will be covered in isCDPipeline.